docs/references/go/overview.mdx

```diff
@@ -5,32 +5,113 @@ description: Learn how to integrate Go into your Clerk application.
 
 # Go
 
-Clerk's Go SDK is a thin wrapper over the Clerk Backend API. Add the following import statement:
+Clerk's [Go SDK](https://github.com/clerk/clerk-sdk-go) is a wrapper over the [Clerk Backend API](https://clerk.com/docs/reference/backend-api).
+
+## Installation
+
+If you're using Go Modules and have a `go.mod` file in your project's root, you can import `clerk-sdk-go` directly:
 
 ```go
-import "github.com/clerkinc/clerk-sdk-go/clerk"
+import (
+    "github.com/clerk/clerk-sdk-go/v2"
+)
 ```
 
-Go doesn't automatically add the module. You can explicitly add it with:
+Alternatively, you can `go get` the package explicitly:
 
-```go
-$ go get github.com/clerkinc/clerk-sdk-go
+```sh filename="terminal"
+go get -u github.com/clerk/clerk-sdk-go/v2
 ```
 
-## `Clerk` client
+## Usage
+
+For details on how to use this module, see the [Go Documentation](https://pkg.go.dev/github.com/clerk/clerk-sdk-go/v2).
+
+The library is organized using a resource-based structure similar to the [Clerk Backend API](https://clerk.com/docs/reference/backend-api).
+Each API supports specific operations, like Create or List. While operations for each resource vary, a similar pattern is applied throughout the library.
+
+To execute API operations, you must configure the library with your Clerk API secret key. Depending on your use case,
+there are two ways to use the library: With or without a client.
+
+For most use cases, the API without a client is a better choice. It requires a minimal setup and provides a more concise API for invoking operations.
+
+However, if you need to operate on multiple Clerk instances from one application, or need more flexibility for tests and mocking,
+you can instantiate multiple API clients with different API keys.
+
+The following examples demonstrate both approaches.
+
+### Usage without a client
+
+If you only use one API key, you can import the packages required for the APIs you need to interact with and call functions for API operations.
```

Comment:
I don't think I understand what this sentence means. @gkats If we reworded it this way, would it still be accurate?

> If you only use one API key, you can import the packages required for the APIs you need. Then you can execute your desired request methods as functions, such as `$resource$.Get()` or `$resource$.Delete()`. The following example demonstrates this process:

Reply:
Yes! Updated.